docs/swagger.yaml

```diff
@@ -79,7 +77,7 @@ paths:
                       turnoverShare: about 25%
                       timeBeforeOrder: 10 days
                       period: once every 6 week
-                      online: 'YES'
+                      isOnlineCourseOffered: true
                       onlineAccessableTime: for 3 months
```

Comment:
Would it help us if we used number of days? Can that help us automate this in some way, some day? If so, we could replace `onlineAccessableTime: for 3 months: for 3 months` with `onlinceAccessibleDays: 90`.

In any case it's "accessible" with i, not e. 🧐 https://www.merriam-webster.com/dictionary/accessible

Reply:
My bad. Fixing this typo 😵‍💫